save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -341,6 +342,26 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                 }
             }
         },
+        updatePermissions = {
```

Comment:
Why is this function passed from here and not created inside `settings` component?

Reply:
Because we want to open error modal window if something goes wrong, so we need to change `ProjectView`'s state. To my mind it is nice enough.

Reply:
Can keep it as is for now